src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -15,15 +15,40 @@
 
 import React, { useState, ChangeEvent } from 'react';
 import { action } from '@storybook/addon-actions';
+import {
+  InteractionTaskArgs,
+  PublicInteractionTask,
+} from 'storybook-addon-performance';
+import { fireEvent, findByText, getByLabelText } from '@testing-library/dom';
```

Comment:
Nit: I'd prefer to use the [`user-event`](https://testing-library.com/docs/ecosystem-user-event/) library everywhere to fire events since it emulates real user behavior.

Reply:
Done in ba968e2